### ISSUE TITLE
Wrapping / spacing issue with < and > operators in conditional statement breaking build

### ISSUE THREAD

**Issue by calda** [nicklockwood/SwiftFormat#1225]
We've ran in to an interesting issue where running SwiftFormat (e.g. just `swiftformat . --indent 2`) on the following code causes it to no longer compile:

### Before

```swift
if
  previousScrollOffsetY < scrollView.contentOffset.y,
  (scrollView.contentOffset.y + scrollView.bounds.size.height) > (threshold * scrollView.contentSize.height)
{ 
  /...
}
```

### After (`swiftformat . --indent 2`)

```swift
if
  previousScrollOffsetY<scrollView.contentOffset.y,
    scrollView.contentOffset.y + scrollView.bounds.size.height>(threshold * scrollView.contentSize.height)
{
  /...
}
```

which no longer compilers.

Not sure exactly what is causing this, but I was able to work around it by wrapping this block in `// swiftformat:disable:all` and `// swiftformat:enable:all` comments.

**Comment by calda** [comment]
seems like it thinks the `<` and `>` tokens are generic parameters on a type named `previousScrollOffsetY`

**Comment by calda** [comment]
This popped up when we enabled the `andOperators` rule. Previously it was written like this, which didn't cause any issues:

```swift
if
  previousScrollOffsetY < scrollView.contentOffset.y
  && (scrollView.contentOffset.y + scrollView.bounds.size.height) > (threshold * scrollView.contentSize.height)
{ 
  /...
}
```

**Comment by nicklockwood** [comment]
Thanks - I think your diagnosis is correct. I'll fix it asap.

**Comment by nicklockwood** [comment]
@calda fixed in 0.49.15
